### ISSUE TITLE
Reliability

### ISSUE THREAD

**Issue by Phillipip** [open-webrtc-toolkit/owt-server#1366]
Is the cluster manager actually capable of automatically recognising failures and automatically distributing the broken nodes to other servers?
Or is the cluster manager only intended for load balancing, not for failover?

**Comment by starwarfan** [comment]
Yes. The cluster manager is only intended for load balancing, not for failover.

**Comment by Phillipip** [comment]
Many thanks for the quick reply
